setuptools_rust/utils.py

```diff
@@ -57,12 +57,32 @@ def rust_features(ext=True, binding=Binding.PyO3):
         raise DistutilsPlatformError(f"unknown Rust binding: '{binding}'")
 
 
-def get_rust_version():
+def get_rust_version(min_version=None):
     try:
         output = subprocess.check_output(["rustc", "-V"]).decode("latin-1")
         return semantic_version.Version(output.split(" ")[1], partial=True)
     except (subprocess.CalledProcessError, OSError):
-        raise DistutilsPlatformError("can't find Rust compiler")
+        raise DistutilsPlatformError(
+            "can't find Rust compiler\n\n"
+            "If you are using an outdated pip version or do not have `wheel` "
+            "installed, it is possible a prebuilt wheel is available for this "
+            "package but pip is not able to install from it. Installing from "
+            "the wheel would avoid the need for a Rust compiler.\n\n"
+            "To update pip and wheel, run:\n\n"
+            "    pip install --upgrade pip wheel\n\n"
```

Comment:
Wheel isn't required on the installer side, pip contains everything required to install wheels. Besides dropping this, I think this is great!

Reply:
👍 thanks, I think maybe in the past it was, at some point? I got confused by some documentation online...